en/advanced_config/land_detector.md

```diff
@@ -30,52 +30,38 @@ Other key parameters that you may need to tune in order to improve landing behav
   
 * [MPC_THR_MIN](../advanced_config/parameter_reference.md#MPC_THR_MIN) - the overall minimum throttle of the system. 
   This should be set to enable a controlled descent.
+* [MPC_LAND_CRWL](../advanced_config/parameter_reference.md#MPC_LAND_CRWL) - the horizontal and vertical speed limit in the last stage of an automatic landing if the system has a distance sensor. Has to be be set larger than LNDMC_Z_VEL_MAX.
 
-
-## Fixed Wing Configuration
-
-The complete set of relevant parameters is available under the [LNDFW](../advanced_config/parameter_reference.md#land-detector) prefix. 
-These two parameters are sometimes worth tuning:
-
-* [LNDFW_AIRSPD_MAX](../advanced_config/parameter_reference.md#LNDFW_AIRSPD_MAX) - the maximum airspeed allowed for the system still to be considered landed. 
-  The default of 8 m/s is a reliable tradeoff between airspeed sensing accuracy and triggering fast enough. 
-  Better airspeed sensors should allow lower values of this parameter.
-* [LNDFW_VEL_XY_MAX ](../advanced_config/parameter_reference.md#LNDFW_VEL_XY_MAX) - the maximum horizontal velocity for the system to be still be considered landed. 
-* [LNDFW_VEL_Z_MAX](../advanced_config/parameter_reference.md#LNDFW_VEL_XY_MAX) - the maximum vertical velocity for the system to be still be considered landed.
-  This parameter can be adjusted to ensure land detection triggers earlier or later on throwing the airframe for hand-launches.
-
-
-<span id="states"></span>
-## Land Detector States
-
-### Multicopter Land Detection
-
-In order to detect landing, the multicopter first has to go through three different states, where each state contains the conditions from the previous states plus tighter constraints.
+n order to detect landing, the multicopter first has to go through three different states, where each state contains the conditions from the previous states plus tighter constraints.
 If a condition cannot be reached because of missing sensors, then the condition is true by default. 
 For instance, in [Acro mode](../flight_modes/acro_mc.md) and no sensor is active except for the gyro sensor, then the detection solely relies on thrust output and time. 
  
-In order to proceed to the next state, each condition has to be true for some predefined time. 
-If one condition fails, the land detector drops out of the current state immediately. 
+In order to proceed to the next state, each condition has to be true for a third of the configured total land detector trigger time [LNDMC_TRIG_TIME](../advanced_config/parameter_reference.md#LNDMC_TRIG_TIME). In case the vehicle is equipped with a distance sensor, but the distance to ground is currently no measurable (usually because it is too large), the trigger time is increased by a factor of 3.
```

Comment:
Fix typo

```suggestion
In order to proceed to the next state, each condition has to be true for a third of the configured total land detector trigger time [LNDMC_TRIG_TIME](../advanced_config/parameter_reference.md#LNDMC_TRIG_TIME).
If the vehicle is equipped with a distance sensor, but the distance to ground is currently not measurable (usually because it is too large), the trigger time is increased by a factor of 3.
```